docs/source/explore/get-started/install.rst

```diff
@@ -53,70 +76,81 @@ To install on Windows, simply double-click on the downloaded installer file to o
-.. For OS X, locate the Vircadia icon in the menu bar. Click on the icon and select 'Quit'.
+You cannot upgrade if Sandbox or the Console is running in the background of your computer. Be sure to quit these applications before upgrading.
 
-.. For more information on the latest releases, see our :doc:`Release Notes <../../release-notes>`.
+For Windows, locate the Vircadia app in your system tray. Right-click on the icon and select 'Quit'. Alternatively, end the 'server-console' background process using the Task Manager.
+For OS X, locate the Vircadia icon in the menu bar. Click on the icon and select 'Quit'.
 
+For more information on the latest releases, see our :doc:`Release Notes <../../release-notes>`.
 
 
 ---------------------------------
 Perform a Clean Install
 ---------------------------------
 
-If you're facing problems when you load Interface and Sandbox, you can try performing a clean install. A clean install removes multiple files and settings that you may need once you install Vircadia again. 
+If you're facing problems when you load Interface and Sandbox, you can try performing a clean install. A clean install removes multiple files and settings that you may need once you install Vircadia again.
 
 .. note:: Ensure that you back up the following files before a clean install: Favorites, Wearables, Sandbox, and Entities. These settings will be deleted during the clean install.
 
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Windows Clean Install
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1. Click on the Start menu and type "Add or Remove Programs" in your Windows search bar.  
-2. Uninstall any versions of Vircadia that are visible (Including any Steam installs). 
-3. Once Vircadia is uninstalled, browse to your %Program Files% directory. Delete all folders related to Vircadia. If you installed through Steam, these folders will be located at ``C:/Program Files(x86)/Steam/steamapps/common``. 
+1. Click on the Start menu and type "Add or Remove Programs" in your Windows search bar.
+2. Uninstall any versions of Vircadia that are visible (Including any Steam installs).
+3. Once Vircadia is uninstalled, browse to your %Program Files% directory. Delete all folders related to Vircadia. If you installed through Steam, these folders will be located at ``C:/Program Files(x86)/Steam/steamapps/common``.
 
-.. warning:: 
+.. warning::
 
-    The next 3 steps will permanently delete your Sandbox content. If you wish to keep this content, copy %AppData%/Local/Project Athena/assignment-client to another location on your computer before proceeding. Repeat for %AppData%/Roaming/Project Athena/assignment-client. 
+    The next 3 steps will permanently delete your Sandbox content. If you wish to keep this content, copy %AppData%/Local/Project Athena/assignment-client to another location on your computer before proceeding. Repeat for %AppData%/Roaming/Project Athena/assignment-client.
```

Comment:
```suggestion
    The next 3 steps will permanently delete your Server Sandbox content. If you wish to keep this content, copy %AppData%/Local/Vircadia/assignment-client to another location on your computer before proceeding. Repeat for %AppData%/Roaming/Vircadia/assignment-client.
```

Reply:
We actually overlooked something here. 
```
Repeat for %AppData%/Roaming/Vircadia/assignment-client.
```
I assume there is a sandbox folder or something this is supposed to point to. @kasenvr do you know what folder this is supposed mean?

Reply:
Nevermind I noticed that one is in Roaming and one is in Local